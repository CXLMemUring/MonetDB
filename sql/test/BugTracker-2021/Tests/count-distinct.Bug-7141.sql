START TRANSACTION;

create table test1 ("id" int, "version" int);
insert into test1 values(1,1),(1,1),(1,2),(1,2),(2,1),(2,2),(2,2),(3,4),(3,4);

SELECT COUNT(distinct "version") FROM test1 GROUP BY "id", "version";

SELECT "id", "version", COUNT(distinct "version") FROM test1 GROUP BY "id", "version";

SELECT "id", "version", COUNT(distinct "version") FROM test1 GROUP BY "id", "version" HAVING COUNT(distinct "version") > 1;

ROLLBACK;
