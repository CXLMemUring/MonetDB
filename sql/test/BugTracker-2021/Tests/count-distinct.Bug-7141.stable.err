stderr of test 'count-distinct.Bug-7141` in directory 'sql/test/BugTracker-2021` itself:


# 08:43:10 >  
# 08:43:10 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-18243" "--port=31137"
# 08:43:10 >  


# 08:43:10 >  
# 08:43:10 >  "Done."
# 08:43:10 >  

