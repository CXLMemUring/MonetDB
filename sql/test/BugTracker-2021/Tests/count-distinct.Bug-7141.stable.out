stdout of test 'count-distinct.Bug-7141` in directory 'sql/test/BugTracker-2021` itself:


# 08:43:10 >  
# 08:43:10 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-18243" "--port=31137"
# 08:43:10 >  

#START TRANSACTION;
#create table test ("id" int, "version" int);
#insert into test values(1,1),(1,1),(1,2),(1,2),(2,1),(2,2),(2,2),(3,4),(3,4);
[ 9	]
#SELECT COUNT(distinct "version") FROM test GROUP BY "id", "version";
% sys.%1 # table_name
% %1 # name
% bigint # type
% 1 # length
[ 1	]
[ 1	]
[ 1	]
[ 1	]
[ 1	]
#SELECT "id", "version", COUNT(distinct "version") FROM test1 GROUP BY "id", "version";
% sys.test1,	sys.test1,	sys.%1 # table_name
% id,	version,	%1 # name
% int,	int,	bigint # type
% 1,	1,	1 # length
[ 1,	1,	1	]
[ 1,	2,	1	]
[ 2,	1,	1	]
[ 2,	2,	1	]
[ 3,	4,	1	]
#SELECT "id", "version", COUNT(distinct "version") FROM test1 GROUP BY "id", "version" HAVING COUNT(distinct "version") > 1;
% sys.test1,	sys.test1,	sys.%1 # table_name
% id,	version,	%1 # name
% int,	int,	bigint # type
% 1,	1,	1 # length
#ROLLBACK;

# 08:43:10 >  
# 08:43:10 >  "Done."
# 08:43:10 >  

